src/figuresClasses.ts

```diff
@@ -1,11 +1,74 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  constructor(
+    public color: string,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
```

Comment:
When using constructor parameters with access modifiers like 'public' or 'private', TypeScript automatically creates and assigns these as properties of the class. Therefore, you don't need to reference 'a', 'b', or 'c' directly in the constructor body. You should use 'this.a', 'this.b', and 'this.c' instead.